src/main/java/io/vertx/core/eventbus/impl/EventBusImpl.java

```diff
@@ -512,6 +522,61 @@ private void unregisterAll() {
     });
   }
 
+  // NOTE: Consider merging the delivery & send context into a general `TransmissionContext`
```

Comment:
DeliveryContext is good for both _send_ and _receive_ , we should remove `SendContext` instead. I know it can break, but this API was never publicly advertised or documented, so I would rather change it now and provide something we can support